src/persist-client/src/lib.rs

```diff
@@ -1067,10 +1065,6 @@ mod tests {
         write.expect_compare_and_append(&data[2..], 3, 4).await;
 
         // Read the snapshot and check that it got all the appropriate data.
-        //
-        // TODO: If we made the SeqNo of the snap and the writes available, we
```

Comment:
I don't think we're going to expose these in the public API. I tried to do this but the semantics get weird around `append`